CMakeLists.txt

```diff
@@ -117,6 +117,27 @@ set(SCAP_WORKBENCH_LINK_LIBRARIES
     Qt5::Widgets Qt5::XmlPatterns
     ${OPENSCAP_LIBRARIES})
 
+# ---------- RPATHS for linking
+
+# see https://cmake.org/Wiki/CMake_RPATH_handling
+
+# use, i.e. don't skip the full RPATH for the build tree
+set(CMAKE_SKIP_BUILD_RPATH FALSE)
```

Comment:
this is the default, why are we setting it to the same value?

Reply:
@mpreisler fixed

Reply:
@mpreisler went ahead and only did rpath handling for Apple as I am not sure if we need to handle it for Linux and windows.